truss/tests/test_truss_util.py

```diff
@@ -16,4 +18,196 @@ def test_max_modified():
         assert t2 > t1
 
 
+def test_load_trussignore_patterns():
+    patterns = path.load_trussignore_patterns(path.TRUSS_IGNORE_PATH)
+
+    assert isinstance(patterns, list)
+    assert "__pycache__/" in patterns
+    assert ".DS_Store" in patterns
+    assert ".git/" in patterns
+
+
+def test_are_dirs_equal(custom_model_truss_dir):
```

Comment:
Not that this is better in general, but for consistency: would be good to mirror target directory under test because that's the convention. So `tests/util/test_path.py`